twa

```diff
@@ -391,8 +392,78 @@ function stage_4_scm_repos {
 # * Fetch the server's cert using `openssl s_client`, check it for known bad suites
 #   * Alternatively, just `nmap --script ssl-enum-ciphers -p 443 $domain`
 
+function stage_5_CAA {
+  verbose "Stage 5: Certification Authority Authorization"
+  verbose "Which Certificate Authorities (CAs) are allowed to issue certificates for the domain"
+    code=`dig +short caa $domain`
+    number_of_lines=`echo $code|wc -l`
+    if [ $number_of_lines -gt 1 ];then
+      INFO "multiple CAAs specified, everyone join the party"
```

Comment:
`INFO` isn't a quantifiable test state, so this should be `verbose` or something similar.

Reply:
agreed